simulator/simulator.go

```diff
@@ -129,6 +129,12 @@ func Fault(msg string, fault types.BaseMethodFault) *soap.Fault {
 	return f
 }
 
+func logPrintf(format string, v ...interface{}) {
```

Comment:
To be honest, when I saw the `logPrintf()` changes above I thought you missed the `.` in between the calls :) 

Suggestion: instead of using this function, why not use a custom io.Writer `Tracer` and pass it to `log.SetOutput` in the condition logic here: https://github.com/vmware/govmomi/blob/df08d4b21fec16b02d2d1b985d316c3762fc3b6c/vcsim/main.go#L104

This would ease extensibility and avoid "forgetting" to call the right log* statement.
Example: https://play.golang.org/p/Y94Z17wxm8V

Reply:
Sounds good, I'll push that change.